docs/semgrep-appsec-platform/notifications.md

```diff
@@ -4,75 +4,16 @@ append_help_link: true
 title: Alerts and notifications
 hide_title: true
 toc_max_heading_level: 4
-description: "Learn how to receive Slack or email alerts about findings and failures, how to receive merge or pull request comments in your CI/CD pipeline, or how to integrate using webhooks."
+description: "Learn how to receive Slack or email alerts about findings and failures and how to integrate using webhooks."
 tags:
     - Semgrep AppSec Platform
     - Team & Enterprise Tier
 ---
 
-
-import EnableAutofix from "/src/components/procedure/_enable-autofix.mdx"
-
-
-
 # Alerts and notifications
 
-Receive notifications in various channels, such as Slack and email, through Semgrep AppSec Platform. This document guides you through setup procedures and Semgrep's deduplication behavior. This ensures that you receive high-signal notifications and alerts in your preferred channels.
-
-Semgrep AppSec Platform can send notifications through the following channels:
-
-| Tool                               | Tier availability |
-| ----                               | ----------------  |
-| [Slack](/semgrep-appsec-platform/slack-notifications)                             | Team & Enterprise  |
-| [Email](/semgrep-appsec-platform/email-notifications)                             | Team & Enterprise  |
-| [Webhooks](/semgrep-appsec-platform/webhooks)                           | Team & Enterprise   |
-
-## Finding available alert and notification channels
```

Comment:
I think it's wise of this PR to remove most of the content in Notifications, except for this bit, because it tells the user "where can I see notifications?" 

I'd probably at least mention that you can "view all available notifications..." etc.